render-products.js

```diff
@@ -12,8 +15,44 @@ export default function renderBug(bug) {
     img.src = bug.image; 
     description.textContent = 'Description: ' + bug.description; 
     price.textContent = 'Price: $' + bug.price;
-    button.innerHTML = 'Add'; 
-    button.addEventListener('click', function(){
+    button.textContent = 'Add';
+    button.value = bug.id;  
+    
+    // button Event Listener
+    button.addEventListener('click', () => {
+        // retrieve existing shopping cart from localStorage. If no cart, we need an empty array. If there is, we need to turn it into a json.parse
+        let cartInLocalStorage = localStorage.getItem('CART'); // set variable to cart in local storage
+        let cart; //creating javascript cart 
```

Comment:
cart already means something else on line 5, when you imported your dummy data.  this is causing the confusion

Reply:
Thanks Sarah, Jacob actually walked me through this issue :) Also this pull request is looking strange and isn't showing my immediate commits that I did today: https://github.com/breeannb/lab-06-products-and-home-page/pull/9/commits/f505e58db6bfcfb1ec84f74a91a62a8d25933bd4